src/components/PostsList.tsx

```diff
@@ -1,86 +1,65 @@
-import React from 'react';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[];
+  isloadingComments?: boolean;
```

Comment:
The `isloadingComments` prop should be renamed to `isLoadingComments` to follow camelCase naming conventions, which is a common practice in JavaScript and TypeScript for readability and consistency.